ui/components/app/advanced-gas-controls/index.scss

```diff
@@ -0,0 +1,44 @@
+.advanced-gas-controls {
+  &__row {
+    margin-bottom: 20px;
+
+    &-heading {
+      display: flex;
+
+      &-tooltip, h6 {
+        display: inline-block;
+      }
+
+      &-detail {
+        text-align: end;
+        flex-grow: 1;
+        font-size: 10px;
+        align-self: center;
+        color: $Grey-500;
```

Comment:
Please use the new color variable names which are those that show up in this map, 
https://github.com/MetaMask/metamask-extension/blob/e144bbd356c5e98cc4960a8b5916c21f56a9caee/ui/css/design-system/colors.scss#L112-L141

and below this line: 
https://github.com/MetaMask/metamask-extension/blob/e144bbd356c5e98cc4960a8b5916c21f56a9caee/ui/css/design-system/colors.scss#L78